samples/bluetooth/direct_test_mode/src/dtm.h

```diff
@@ -15,6 +15,180 @@
+ * supported.
+ */
+#define LE_TEST_SUPPORTED_CTE_LENGTH 0x10
+
+/* DTM command parameter: Set PHY for future packets to use 1MBit PHY.
+ * Minimum parameter value.
+ */
+#define LE_PHY_1M_MIN_RANGE 0x04
+
+/* DTM command parameter: Set PHY for future packets to use 1MBit PHY.
+ * Maximum parameter value.
+ */
+#define LE_PHY_1M_MAX_RANGE 0x07
+
+/* DTM command parameter: Set PHY for future packets to use 2MBit PHY.
+ * Minimum parameter value.
+ */
+#define LE_PHY_2M_MIN_RANGE 0x08
+
+/* DTM command parameter: Set PHY for future packets to use 2MBit PHY.
+ * Maximum parameter value.
+ */
+#define LE_PHY_2M_MAX_RANGE 0x0B
+
+/* DTM command parameter: Set PHY for future packets to use coded PHY with S=8.
+ * Minimum parameter value.
+ */
+#define LE_PHY_LE_CODED_S8_MIN_RANGE 0x0C
+
+/* DTM command parameter: Set PHY for future packets to use coded PHY with S=8.
+ * Maximum parameter value.
+ */
+#define LE_PHY_LE_CODED_S8_MAX_RANGE 0x0F
+
+/* DTM command parameter: Set PHY for future packets to use coded PHY with S=2.
+ * Minimum parameter value.
+ */
+#define LE_PHY_LE_CODED_S2_MIN_RANGE 0x10
+
+/* DTM command parameter: Set PHY for future packets to use coded PHY with S=2.
+ * Maximum parameter value.
+ */
+#define LE_PHY_LE_CODED_S2_MAX_RANGE 0x13
+
+/* Mask of the CTE type in the CTEInfo. */
+#define LE_CTE_TYPE_MASK 0x03
+
+/* Position of the CTE type in the CTEInfo. */
+#define LE_CTE_TYPE_POS 0x06
+
+/* Mask of the CTE Time in the CTEInfo. */
+#define LE_CTE_CTETIME_MASK 0x1F
+
+/* DTM command parameter: Minimum supported CTE length in 8 us units. */
+#define LE_CTE_LENGTH_MIN 0x02
+
+/* DTM command parameter: Maximum supported CTE length in 8 us units. */
+#define LE_CTE_LENGTH_MAX 0x14
+
+/* DTM command parameter: Mask of the Antenna Number. */
+#define LE_ANTENNA_NUMBER_MASK 0x3F
+
+/* DTM command parameter: Mask of the Antenna switch pattern. */
+#define LE_ANTENNA_SWITCH_PATTERN_MASK 0x80
+
+/* Minimum antenna number. */
+#define LE_TEST_ANTENNA_NUMBER_MIN 0x01
+
+/* Maximum antenna number. */
+#define LE_TEST_ANTENNA_NUMBER_MAX 0x4B
+
+/* DTM command parameter: Minimum supported transmit power level. */
+#define LE_TRANSMIT_POWER_LVL_MIN -127
+
+/* DTM command parameter: Maximum supported transmit power level. */
+#define LE_TRANSMIT_POWER_LVL_MAX  20
+
+/* DTM command parameter: Set minimum transmit power level. */
+#define LE_TRANSMIT_POWER_LVL_SET_MIN 0x7E
+
+/* DTM command parameter: Set maximum transmit power level. */
+#define LE_TRANSMIT_POWER_LVL_SET_MAX 0x7F
+
+/* Position of power level in the DTM power level set response. */
+#define LE_TRANSMIT_POWER_RESPONSE_LVL_POS (0x01)
+
+/* Mask of the power level in the DTM power level set respose. */
+#define LE_TRANSMIT_POWER_RESPONSE_LVL_MASK (0x1FE)
+
+/* Maximum power level bit in the power level set response. */
+#define LE_TRANSMIT_POWER_MAX_LVL_BIT BIT(0x0A)
+
+/* Minimum power level bit in the power level set response. */
+#define LE_TRANSMIT_POWER_MIN_LVL_BIT BIT(0x09)
```

Comment:
As you can see, I refactored this DTM code part from nRF5 SDK by grouping logically defines that correspond to the same parameter/command / etc. Could you do the same here? I think the code is more readable when you use enum groups instead of a big block of defines.

Reply:
Ok I grouped some defines into enum types. I left the ones for which I don't think it would make sense